website/tests/test_models.py

```diff
@@ -33,4 +33,70 @@ def then_the_user_has_expected_values(self):
         self.assertEqual(self.user.last_name, self.LAST_NAME)
         self.assertEqual(self.user.email_address, self.EMAIL_ADDRESS)
         self.assertEqual(self.user.password, self.PASSWORD)
-        self.assertEqual(self.user.salt, self.SALT)
\ No newline at end of file
+        self.assertEqual(self.user.salt, self.SALT)
+
+
+class NotificationTest(TestCase):
+    ID = "1234ABCD"
+    NAME = "My Notification"
+    NOTIFICATION_GROUP = "My Group"
+    ADDRESS = "123 Main St."
+    CITY = "San Antonio"
+    STATE = "TX"
+    ZIPCODE = "12345"
+    DATE = "Jan 1, 1970"
+    TIME = "12:00 AM"
+
+    notification = Notification()
+
+    def test_notification_creation(self):
+        self.given_user_has_user_account_is_created()
+        self.then_the_notification_has_expected_values()
+
+    def given_user_has_user_account_is_created(self):
+        self.notification = Notification.objects.create(
+            id=self.ID,
+            name=self.NAME,
+            address=self.ADDRESS,
+            city=self.CITY,
+            state=self.STATE,
+            zipcode=self.zipcode,
+            date=self.DATE,
+            time=self.TIME
+        )
+
+    def then_the_notification_has_expected_values(self):
+        self.assertTrue(isinstance(self.notification, Notification))
+        self.assertEqual(self.notification.id, self.ID)
+        self.assertEqual(self.notification.name, self.NAME)
+        self.assertEqual(self.notification.address, self.ADDRESS)
+        self.assertEqual(self.notification.city, self.CITY)
+        self.assertEqual(self.notification.state, self.STATE)
+        self.assertEqual(self.notification.zipcode, self.ZIPCODE)
+        self.assertEqual(self.notification.date, self.DATE)
+        self.assertEqual(self.notification.time, self.TIME)
+
+
+class NotifcationGroupTest(TestCase):
+    NAME = "GROUP TEST"
+    EMAIL = "test@email.com"
+    EMAIL_LIST = "{\"emailList\":[{\"email\":\"email1@email.com\"},{\"email2\":\"email1@email.com\"}]}"
+
+    notification = Notification()
+
+    def test_group_creation(self):
+        self.given_user_has_user_account_is_created()
+        self.then_the_group_has_expected_values()
+
+    def given_user_has_user_account_is_created(self):
```

Comment:
These method names should be updated to reflect that it's testing notification group and not user